plugins/plugin-java/che-plugin-java-ext-lang-server/src/main/java/org/eclipse/che/plugin/java/server/rest/ClasspathService.java

```diff
@@ -19,35 +30,24 @@
 import org.eclipse.jdt.internal.core.JavaModel;
 import org.eclipse.jdt.internal.core.JavaModelManager;
 
-import javax.ws.rs.GET;
-import javax.ws.rs.Path;
-import javax.ws.rs.Produces;
-import javax.ws.rs.QueryParam;
-import javax.ws.rs.core.MediaType;
-import java.util.ArrayList;
-import java.util.List;
-
-import static java.util.Collections.emptyList;
-
 /**
  * Service for getting information about classpath.
  *
  * @author Valeriy Svydenko
  */
 @Path("java/classpath/")
-public class ClasspathService {
+public class ClasspathService implements ClasspathServiceInterface {
 
```

Comment:
I assume you've added this interface for testing purposes.
From our experience this approche  has some disadvantages.
1. Inheritance in Rest services is quite unusual and we are trying to avoid it in our code.
2. Name of interface is syntetic, it show that it was made for some reason, but not for real inheritance.
3. Mockitng of rest service is quite rare too.

If you do realy want to test Rest service I recomend you to take a look on @Listeners(value = {EverrestJetty.class, MockitoTestNGListener.class})
For example 
https://github.com/eclipse/che/blob/master/wsmaster/che-core-api-user/src/test/java/org/eclipse/che/api/user/server/UserServiceTest.java
https://github.com/eclipse/che/blob/master/wsmaster/che-core-api-workspace/src/test/java/org/eclipse/che/api/workspace/server/WorkspaceServiceTest.java